code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -35,11 +35,11 @@
 	load_method = MAGAZINE
 	magazine_type = /obj/item/ammo_magazine/m10mm
 	allowed_magazines = list(/obj/item/ammo_magazine/m10mm)
+	projectile_type = /obj/item/projectile/bullet/pistol/medium
 	auto_eject = 1
 	auto_eject_sound = 'sound/weapons/smg_empty_alarm.ogg'
 
-//	requires_two_hands = 1
-	one_handed_penalty = 2
+//	one_handed_penalty = 15

```

Comment:
2*15=30?

Reply:
This is the submachine gun Mercs get, and I don't want it to be unusable with the e-shield.